server/src/services/donatedItemService.ts

```diff
@@ -0,0 +1,30 @@
+import prisma from '../prismaClient'; 
+
+// export async function validateDonatedItem(donatedItemId: number) {
```

Comment:
Remember to remove commented out code before you submit

Reply:
I agree with Joseph

Reply:
@saikiran0405 please remove the unwanted comments